pyproject.toml

```diff
@@ -31,13 +31,13 @@ requires-python = ">=3.9,<4"
 dependencies = [
     "cloudpickle >= 1.5.0",
     "holidays >= 0.17",
-    "numpy >= 1.21.0",
+    "numpy >= 1.25.0",
```

Comment:
Over slack you mentioned `1.25.1` - happy to doo whichever, but just wanted to point it out!